ci/publish_github_release.sh

```diff
@@ -34,4 +34,9 @@ tar -xvjpf /tmp/ghrelease.tar.bz2 -C /tmp
 cp /tmp/bin/linux/amd64/github-release /usr/local/bin/ghrelease
 chmod +x /usr/local/bin/ghrelease
 
-ghrelease upload --tag "${CIRCLE_TAG:-}" --name "envoy-linux-amd64" --file "${ENVOY_SRCDIR}/build_release_stripped/envoy"
+ghrelease upload \
+    --user "envoy-bot" \
```

Comment:
How is this different from the env vars? And shouldn't this be "envoyproxy"? Also, what about creds leaking?

Reply:
I was unable to see what the env vars are set to, so this makes it explicit.

I don't know if the user should be "envoyproxy". From looking at the [github-release code](https://github.com/aktau/github-release/blob/f2ebbf795aa4ab88b13c940c7df6807ce79c2b08/github-release.go#L97-L99) I assume repo should be "envoy" and user should be the user, in this case "envoy-bot". These are assumptions and I may be wrong.

Reply:
creds leaking are 100% a problem and is not addressed by this.